corehq/apps/users/views/__init__.py

```diff
@@ -454,13 +457,30 @@ def user_roles(self):
             key=lambda role: role.name if role.name else u'\uFFFF'
         ))
 
+        show_es_issue = False
         # skip the admin role since it's not editable
         for role in user_roles[1:]:
-            role.hasUsersAssigned = bool(role.ids_of_assigned_users)
+            try:
+                role.hasUsersAssigned = bool(role.ids_of_assigned_users)
+            except TypeError:
+                # when query_result['hits'] returns None due to an ES issue
+                show_es_issue = True
             role.has_unpermitted_location_restriction = (
                 not self.can_restrict_access_by_location
                 and not role.permissions.access_all_locations
             )
+        raise Exception()
```

Comment:
This will unconditionally raise an exception

Reply:
whoops. that was supposed to be removed. was for testing. thanks